spinta/types/helpers.py

```diff
@@ -29,3 +39,27 @@ def set_dtype_backend(dtype: DataType):
         dtype.backend = backends[dtype.backend]
     else:
         dtype.backend = dtype.prop.model.backend
+
+
+def check_model_name(context: Context, model: Model):
+    config: Config = context.get('config')
+    if config.check_names:
+        name = model.name
+        if name not in ['_ns', '_txn', '_schema', '_schema/version']:
+            if upper_camel_case.match(name) is None:
```

Comment:
Check model name and namespace separately. For example:

    a/b/c/Model

Here `a/b/c` is a namespace and should be checked with `check[Context, Namespace]`, `Model` is a model name and should be checked separately with `check[Context, Model]`.